README.md

```diff
@@ -127,7 +127,7 @@ family_name = profile.family_name.value
 full_name = profile.full_name.value
 phone_number = profile.phone_number.value
 date_of_birth = profile.date_of_birth.value
-is_age_verified = profile.is_age_verified.value
+age_verified = profile.age_verified.value
```

Comment:
Is the current version exposing `is_age_verified` ? If so, this is a breaking change and it cannot go out like this.
You need to deprecate `is_age_verified` instead and add `age_verified`.

Reply:
Was just committing as you commented: I have kept the `is_age_verified` for activity_details, and we now use `age_verified` for the profile object. I've tested this on the old version of the Django example project too to check it works. Do you think we should add anything to the API coverage? Or it would just be sufficient to include this information in the release notes?